posts/2022-01-17-why-we-combined-graphql-and-a-serverless-distributed-ledger.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Why We Combined GraphQL and a Serverless Distributed Ledger"
+description: "Making Strategic Decisions to Enable a Platform"
+date: '2022-01-18'
+categories:
+- concepts
+authors:
+- Brian McNamara
+---
+
+Vendia Share is a SaaS offering that takes away the toil required to share data with partners in a controlled, secure, and ledgered manner. We do this by allowing customers to create Universal Applications (Unis) and inviting their partners to participate.
+
+Each participant in a Uni has cloud resources created on their behalf. One of the resources is a managed, scalable, resilient, and secure GraphQL API that allows partners to create web, mobile, and backend applications to read data from and write data to a Uni.
+
+While technical readers may be familiar with the concept of a REST API, there may be some among you who are not as familiar with GraphQL. For those of you with GraphQL experience, Vendia Share does take a non-traditional approach in some areas as well.  In this post, we'll dive a bit deeper and explore GraphQL and look at why Vendia selected GraphQL as its primary interface. Through this post, you'll learn more about GraphQL and how it can improve your development experience with Vendia Share.
+
+# What is GraphQL?
+
+[GraphQL was created by Meta / Facebook in 2012](https://engineering.fb.com/2015/09/14/core-data/graphql-a-data-query-language/) as a way to solve problems encountered when rebuilding their native mobile applications. The team was frustrated with the differences between the data they wanted to use in their applications and the server queries that were required and was challenged to pull in data from many disparate sources. They saw the importance of seeing data not just as a resource but as a graph.
```

Comment:
```suggestion
[GraphQL was created by Meta / Facebook in 2012](https://engineering.fb.com/2015/09/14/core-data/graphql-a-data-query-language/) as a way to solve problems encountered when rebuilding their native mobile applications. The team was frustrated with the differences between the data they wanted to use in their applications and the server queries that were required and was challenged to pull in data from many disparate sources. They saw the importance of treating data not just as a resource but as a graph.
```

Reply:
Attempting to avoid "saw" and "seeing" in the same sentence.  Take or leave as you see fit.